Project.toml

```
@@ -14,7 +15,8 @@ SemidefiniteModels = "169818f4-1a3d-53bf-95b3-11177825b1e3"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-BinDeps = "0.8, 0.9, 1"
+BinaryProvider = "0.5"
```

You may want to require v0.5.9 specifically:
```suggestion
BinaryProvider = "0.5.9"
```

---

I think it may be good to bump BP to 0.6.0

---

That version requires Julia v1.3, which is not very helpful here.  Yes, history of BinaryProvider is a bit messy :slightly_smiling_face:

---

Ah - that is a bit messy.